{
  EventRecord.here(CallbackListenerSequence.class,EventType.CALLBACK,"fire(" + response.getClass().getName() + ")").extreme();
  for (  Callback<? super R> cb : this.successCallbacks) {
    try {
      EventRecord.here(this.getClass(),EventType.CALLBACK,"" + cb.getClass(),"fire(" + response.getClass().getCanonicalName() + ")").extreme();
      cb.fire(response);
    }
 catch (    Exception t) {
      this.LOG.error("Exception occurred while trying to call: " + cb.getClass() + ".apply( "+ t.getMessage()+ " )");
      this.LOG.error(t,t);
    }
  }
}
